src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -20,8 +20,10 @@ internal static class FatalError
         private static Action<Exception>? s_fatalHandler;
         private static Action<Exception>? s_nonFatalHandler;
 
+#pragma warning disable IDE0052 // Remove unread private members - We want to hold onto last exception to make investigation easier
```

Comment:
@dotnet/roslyn-compiler for changes to few files shared between Compiler and Workspaces layer. Ideally, this PR would not have any changes under `src\Compilers`, but they do due to https://github.com/dotnet/roslyn/issues/43371#issuecomment-614124642.